### PR TITLE
Initial recipe for pygeo/easytest

### PR DESCRIPTION
**Pull request by bulli92**
not sure about skip options

### CODE REVIEW

File: recipes/example/meta.yaml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (synapticarbors) [inline recipes/example/meta.yaml]: Please revert all changes made to `example/meta.yaml`

File: recipes/easytest/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,71 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (synapticarbors) [inline recipes/easytest/meta.yaml]: Please delete all comments.

File: recipes/easytest/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set name = "easytest" %}
+{% set version = "0.1.3" %}
+{% set sha256 = "bfecf6663b14bd5418e2d36b615ccdf7" %}
```
Comment (synapticarbors) [inline recipes/easytest/meta.yaml]: I believe the sha256 has should be `5e31be6e32f2672e78d59ef40251c7d6b342a4c27793eef0006d3af36d73da59`

File: recipes/easytest/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+  number: 0
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  # By default, the package will be built for Python 2.7, 3.4, and above, for all major OSs.
+  # Add the line "skip: True  # [py<34]" (for example) to limit to Python 3.4 and newer, or "skip: True  # [not win]" to limit to Windows.
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [py>27]
+  skip: True  # [win]
+
+requirements:
+  build:
```
Comment (synapticarbors) [inline recipes/easytest/meta.yaml]: It looks like you need `numpy` here since the `setup.py` imports it.

File: recipes/easytest/meta.yaml (hunk 3 above)
Comment (bulli92) [inline recipes/easytest/meta.yaml]: This is actually the md5 key taken from pipit like described in the documentation.

Comment (synapticarbors) [inline recipes/easytest/meta.yaml]: I'm unsure what "pipit" is or which documentation you are referring to, but you cannot use an md5 hash in the place of a sha256 hash. Please refer to https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L7-L10 or in the case of packages where the source is pypy.io, you can just click on the "sha256" link on the package page: 

https://pypi.org/project/easytest/#files

Comment (bulli92) [inline recipes/easytest/meta.yaml]: Sorry, was a typo, I mean pypi. In the old layout I could only find the md5 key.
Using your link I could find the sha256 now. Thanks!

**Comment by bulli92** [comment]
o.k., have reverted the original example, updated dependencies and cleaned comments.
Hope it does the job now.

**Comment by bulli92** [comment]
o.k. doesn't seem to works till because of numpy dependncies. Travis complains about

  File "/opt/conda/lib/python3.5/site-packages/conda/resolve.py", line 605, in get_pkgs
    raise NoPackagesFoundError([(ms,)])
conda.exceptions.NoPackagesFoundError: Package missing in current linux-64 channels: 
  - numpy 1.12


What confuses me is 
a) who should I properl yspecify the numpy dependency?
b) I assumed that I have skipped anything of python 3.x in my recipie specification. Obviously I did wrong as the error is caused within python3.5 environment.

Thus I don't really know how to change things that they work. Sorry for these questions, but this is my first recipie for conda-forge.

**Comment by synapticarbors** [comment]
I do not think that you need to specify the numpy version. I don't see any indication that the package relies on a particular version, so I would just remove the `1.12`. More generally though, if you're building a package that needs a different build for different versions of numpy, you can specify the dependency as `numpy x.x` and then conda-forge will build it against the matrix of numpy versions:

https://conda.io/docs/building/meta-yaml.html#run

**Comment by synapticarbors** [comment]
Looks good to me, although I have one final question -- Is there are reason to skip the windows build?

**Comment by bulli92** [comment]
never tried the package on windows so far. This is why I skipped it. Same is for python 3.x
Planned to do that for a later version and run tests in different environments using travis.ci

**Comment by synapticarbors** [comment]
It seems like there would be no harm in removing the skip lines and letting the conda-forge CIs check to see if everything passes for windows and py3.x, but it's fine to defer that for now, especially since you are the primary author of the library. You can do that as a PR in the feedstock.

**Pull request merged**